packages/e2e/tests/cards/errorPanel/errorPanel.kcpCard.visible.test.js

```diff
@@ -12,7 +12,7 @@ const INVALID_TAX_NUMBER = LANG['creditCard.taxNumber.invalid'];
 
 const cardPage = new CardComponentPage();
 
-fixture`Testing kcpCard, with holderName, error panel`
```

Comment:
I guess you forgot to remove the `only` :)

Reply:
guess so! Fixed